wp-admin/js/widgets/media-widgets.js

```diff
@@ -111,7 +111,9 @@ wp.mediaWidgets = ( function( $ ) {
 
 			// Re-render the preview when the attachment changes.
 			control.selectedAttachment = new wp.media.model.Attachment( { id: 0 } );
+			control.renderPreview = _.debounce( control.renderPreview );
```

Comment:
Gotcha, so this saves one render in instances where you are changing media/url.  Looks like the preview is rendered twice upon initial load as well when the model is being updated.  While playing with this I noticed `renderPreview` is called on each keystroke when editing the widget title currently so perhaps that is a good candidate for a `_.throttle` if we are concerned about too many renders.